code/modules/vending/_vending.dm

```diff
@@ -861,6 +870,12 @@
 /atom/movable/proc/post_tilt()
 	return
 
+/atom/movable/proc/can_be_vendorcrushed(atom_target)
+	if !(is_type_in_list(atom_target, vendor_nocrush))//make sure its not in the list of "uncrushable" stuff
+		if (atom_target.uses_integrity && !(atom_target.invisibility > SEE_INVISIBLE_LIVING) //allow ninjas, etc to be crushed in cloak
+			return TRUE //yep its crushable
+	return FALSE //turns out its not crushable.
```

Comment:
if you make it a static list in stead of a global, be sure to replace line 874 with just `vendor_uncrushable_items`

Reply:
> this... is really weird. your parenthesis are completely unbalanced. have a complimentary rewrite. we should also be typing `atom_target` as well because there's no way this works. this is just rewritten to be a bit cleaner with early returns.

Okay! I considered using early returns, but wasn't sure if having duplicate code like that was frowned upon, due to being extra.

Reply:
> this... is really weird. your parenthesis are completely unbalanced. have a complimentary rewrite. we should also be typing `atom_target` as well because there's no way this works. this is just rewritten to be a bit cleaner with early returns.

For some reason, the commit I made yesterday that rebalanced a missing parenthesis never got sent through VS Code. Yours is def cleaner though, happy to go with that.